core/io/image.cpp

```diff
@@ -2645,6 +2645,25 @@ Error Image::compress(CompressMode p_mode, CompressSource p_source, ASTCFormat p
 Error Image::compress_from_channels(CompressMode p_mode, UsedChannels p_channels, ASTCFormat p_astc_format) {
 	ERR_FAIL_COND_V(data.is_empty(), ERR_INVALID_DATA);
 
+	// RenderingDevice only.
+	if (OS::get_singleton()->get_current_rendering_method() == "forward_plus") {
+		switch (p_mode) {
+			case COMPRESS_BPTC: {
+				// BC7 is unsupported currently.
+				if (format >= FORMAT_RH && format <= FORMAT_RGBE9995) {
+					ERR_FAIL_NULL_V(_image_compress_bptc_rd_func, ERR_UNAVAILABLE);
+					_image_compress_bptc_rd_func(this, p_channels);
+					return OK;
+				}
+
+				break;
```

Comment:
```suggestion
			} break;
```
We normally put the `break` after the bracket